oak/client/application_client.h

```diff
@@ -90,25 +102,11 @@ class ApplicationClient {
   // on additional metadata or specific time-related policies.
   //
   // See https://grpc.io/docs/guides/auth/.
-  static std::shared_ptr<grpc::Channel> CreateChannel(std::string addr) {
+  static std::shared_ptr<grpc::Channel> CreateChannel(
+      std::string addr, std::shared_ptr<grpc::CallCredentials> call_credentials) {
```

Comment:
drive-by thought: Why not create a `CreateChannel(addr)` overload that forwards on to `CreateChannel(addr, private_authorization_bearer_token_call_credentials())`?  That would reduce the churn on `examples/*/client/*cc`.